src/test/java/org/folio/edge/sip2/api/MainVerticleTests.java

```diff
@@ -79,8 +81,38 @@ public void canStartMainVericleInjectingSip2RequestHandlers(Vertx vertex,
   @Test
   public void cannotCheckoutWithInvalidCommandCode(Vertx vertex,
       VertxTestContext testContext) throws Throwable {
-    callService("blablabalb", testContext, vertex, result -> {
+      callService("blablabalb", testContext, vertex, result -> {
```

Comment:
I don't think this indentation is correct. It shouldn't be at the same level as the lamda block on the next line nor should it line up with the method argument list the line above. I think the original indentation is correct. Is checkstyle requiring this indentation?